Assets/Thirdweb/Examples/Scripts/Prefab_NFTLoader.cs

```diff
@@ -0,0 +1,106 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using Thirdweb;
+
+[Serializable]
+public struct NFTQuery
+{
+    public List<NFTQuery_Single> singleQueries;
+    public List<NFTQuery_Multi> multiQueries;
+    public List<NFTQuery_Owned> ownedQueries;
```

Comment:
can you show me a code snippet on how ppl would use this? have a feeling it's gonna be awkward to use.

Intuitively i think I would expect something like this:
```
// pseudo code
var nfts = await contract.ERC721.GetAll();
for (nft in nfts) {
  Instantiate(myNFTPrefab, ...)
}
```

that would be the minimal setup i think, but quite straightforward. That said i don't know if we can instantiate prefabs with parameters like that?

then intuitively my next thought is - can there be a "NFTGrid" prefab or "NFTList" prefab that takes in a List<NFT>

Reply:
They would drag and drop the NFT Loader prefab to load multiple NFTs they want is the idea, think to make a shop or a marketplace view. 

For the Prefab_NFT they can instantiate that through a script that takes it as a reference and Load it like I did in the NFT Loader script, if they want to use it separately